src/components/circular-progress/circular-progress.css

```diff
@@ -0,0 +1,76 @@
+:host {
+  display: block;
+}
+
+
+
+@-webkit-keyframes circular-rotate {
```

Comment:
Don't need to use `-webkit-` prefix because our bundler will add prefixes after build